modules/installation-osp-creating-bootstrap-machine.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+[id="installation-osp-creating-bootstrap-machine_{context}"]
+= Creating the bootstrap machine
+
+The playbook sets the allowed address pairs on each port attached to our OpenShift nodes.
+
+Since the keepalived-managed IP addresses are not attached to any specific server, Neutron would block their traffic by default. By passing them to --allowed-address the traffic can flow freely through.
+
+An additional Floating IP is also attached to the bootstrap port. This is not necessary for the deployment (and we will delete the bootstrap resources afterwards). However, if the bootstrapping phase fails for any reason, the installer will try to SSH in and download the bootstrap log. That will only succeed if the node is reachable (which in general means a floating IP).
+
+.Prerequisites
+* The infrastructure ID from the installation program's metadata file is set as an environment variable (`$INFRA_ID`)
+* The `inventory.yaml` and `common.yaml` Ansible playbooks
+** If you need these files, copy them from *Creating network resources*
+
+.Procedure
+
+. Insert the following content into a local file called `03_bootstrap.yaml`:
++
+[source,yaml]
+----
+# Required Python packages:
+#
+# ansible
+# openstacksdk
+# netaddr
+
+- import_playbook: common.yaml
+
+- hosts: all
+  gather_facts: no
+
+  tasks:
+  - name: 'Create the bootstrap server port'
+    os_port:
+      name: "{{ os_port_bootstrap }}"
+      network: "{{ os_network }}"
+      security_groups:
+      - "{{ os_sg_master }}"
+      allowed_address_pairs:
+      - ip_address: "{{ os_subnet_range | next_nth_usable(5) }}"
+      - ip_address: "{{ os_subnet_range | next_nth_usable(6) }}"
+
+  - name: 'Set bootstrap port tag'
+    os_tag:
+      resource: "port"
+      name: "{{ os_port_bootstrap }}"
+      tags: "{{ [cluster_id_tag] }}"
+
+  - name: 'Create the bootstrap server'
+    os_server:
+      name: "{{ os_bootstrap_server_name }}"
+      image: "{{ os_image_rhcos }}"
+      flavor: "{{ os_flavor_master }}"
+      userdata: "{{ lookup('file', os_bootstrap_ignition) | string }}"
+      auto_ip: no
+      nics:
+      - port-name: "{{ os_port_bootstrap }}"
+
+  - name: 'Create the bootstrap floating IP'
+    os_floating_ip:
+      state: present
+      network: "{{ os_external_network }}"
+      server: "{{ os_bootstrap_server_name }}"
+
+----
+
+. On a command line, run the playbook:
++
```

Comment:
Before you run the playbooks you need to copy all the json files created in the sections above to the same location as the plabooks. 

so before running the command below you have to do something like: 

$ cp <path_to_directory_where_install-config.yaml_file_is>/*.json ./

Because the playbooks look for metadata.json, master-index-*.json files in same directory where the playbooks are.